conans/client/downloaders/file_downloader.py

```diff
@@ -102,12 +103,13 @@ def get_total_length():
 
         try:
             total_length = get_total_length()
-            if total_length > 1000000:  # 10 Mb
-                from conan.tools.files.files import _human_size
-                hs = _human_size(total_length)
+            is_large_file = total_length > 10000000  # 10 Mb
```

Comment:
Seems like this was also missing a 0, oops!

Same thing, I know it's extreme nitpick but:

```suggestion
            is_large_file = total_length > 10000000  # 10 MB
```